example/example.js

```diff
@@ -77,7 +77,6 @@ var Example = React.createClass({
         },
         timePrefs: {
           timezoneAware: false,
-          // timezoneAware: true,
```

Comment:
Can you add this back? I find it useful to keep around for toggling back and forth by commenting in/out.